Changelog for Dec 20 (#3673)
timescale/docs: about/changelog.md

* **More Flexible MERGE Operations**. The MERGE command now includes a RETURNING clause, making it easier to track and work with modified data. You can now also update views using MERGE, unlocking new use cases for complex queries and data manipulation.
  
### Submit feature requests from Timescale Console
You can now submit feature requests directly from Console and see the list of feature requests you have made. Just click on Feature Requests on the right sidebar.
All feature requests are automatically published to the [Timescale Forum](https://www.timescale.com/forum/c/cloud-feature-requests/39) and are reviewed by the product team, providing more visibility and transparency on their status as well as allowing other customers to vote for them.

![Submit a feature request in Timescale Console](https://s3.us-east-1.amazonaws.com/assets.timescale.com/docs/images/submit-feature-request.png)

### Postgres LiveSync (Alpha release)
We have built a new solution that helps you continuously replicate all or some of your Postgres tables directly into Timescale.

> It allows you to keep your current Postgres instance (ex: RDS) as your primary database and easily offload your real-time analytical queries to Timescale to boost their performance.

Reviewer: ```suggestion
[Livesync](https://docs.timescale.com/migrate/latest/livesync/) allows you to keep a current Postgres instance such as RDS as your primary database, and easily offload your real-time analytical queries to Timescale to boost their performance. If you have an questions or feedback, talk to us in [#livesync in Timescale Community](https://app.slack.com/client/T4GT3N2JK/C086NU9EZ88).
```

> [Livesync](https://docs.timescale.com/migrate/latest/livesync/) allows you to keep a current Postgres instance such as RDS as your primary database, and easily offload your real-time analytical queries to Timescale Cloud to boost their performance. If you have an questions or feedback, talk to us in [#livesync in Timescale Community](https://app.slack.com/client/T4GT3N2JK/C086NU9EZ88).

Reviewer 2: ```suggestion
[Livesync](https://docs.timescale.com/migrate/latest/livesync/) allows you to keep a current Postgres instance such as RDS as your primary database, and easily offload your real-time analytical queries to Timescale Cloud to boost their performance. If you have any questions or feedback, talk to us in [#livesync in Timescale Community](https://app.slack.com/client/T4GT3N2JK/C086NU9EZ88).
```

> This is just the beginning—you'll see more from LiveSync in 2025!

Reviewer 2: ```suggestion
This is just the beginning—you'll see more from Livesync in 2025!
```
